public/static/js/task.js

```diff
@@ -586,7 +586,25 @@ mciModule.controller('TaskLogCtrl', ['$scope', '$timeout', '$http', '$location',
     return '[' + timestamp + '] '
   }
 
+  var isUnfinished = function(status) {
+    switch (status) {
```

Comment:
Similarly, here, is there a shorter list of statuses that _are_ finished?

Reply:
Initially I thought this would be shorter :(